tools/doc/README.md

```diff
@@ -105,4 +105,3 @@ This event is emitted on instances of SomeClass, not on the module itself.
 * Classes have (description, Properties, Methods, Events)
 * Events have (list of arguments, description)
 * Methods have (list of arguments, description)
-* Properties have (type, description)
```

Comment:
If the entire bullet list was sorted this would not have happened, can you sort the entire list as you are removing the duplicate?